community/2024-04-03-community-meeting.mdx

```diff
@@ -31,7 +31,7 @@ metadata:
   annotations:
     version: v0.0.1
     description: 'HTTP hello world demo in Rust, using the WebAssembly Component Model and WebAssembly Interfaces Types (WIT)'
-    experimental: true
+    experimental: 'true'
```

Comment:
```suggestion
    experimental: "true"
```

Can we get consistent quotes? it's wild that we use both `'` and `"` all over the place

Reply:
I was being consistent on a per example basis based on how the `description` is set, we aren't being consistent there across the board, so I figured it was more important to match that style

Reply:
I propose once we decide on the style we want to follow, we fix this across all of the examples with the new style in mind.